perffile/format.go

```diff
@@ -950,6 +951,16 @@ func (r *RecordTextPoke) Type() RecordType {
 	return RecordTypeTextPoke
 }
 
+type RecordAuxOutputHardwareID struct {
+	RecordCommon
+
+	ID uint64
```

Comment:
Ugh, perf. It might be good to include a bit of the commentary from the commit message of 8b8ff8cc3b8155c18162e8b1f70e1230db176862 to explain what on earth this ID identifies.